### ISSUE TITLE
Integration with conference organization software

### ISSUE THREAD

**Issue by XVilka** [opencollective/opencollective#2422]
There are various conference organization platforms, and many big open source projects use some of them for setting up a developers/users conference revolving about this platform. It is a very rough idea, but you might think about the ways of this integration for 'Events' feature:
- [Indico](https://getindico.io/)
- [OSEM](https://osem.io/)
- [Eventbrite](https://www.eventbrite.hk/)
- [Pretalx](https://pretalx.com/p/about/)
- many others

**Comment by piamancini** [comment]
Yes, we discussed this recently and we agree that integrating is a better option than having to maintain our own event feature which is not our focus. cc @alanna for the future.

**Comment by alanna** [comment]
It would be great to integrate with third party providers like these, as we'll never be able to provide advanced functionality for events and ticketing like they do. We'd much prefer to let an events tool do what its good at so we can stay focused on our piece (finances).

Currently we 'integrate' with third party providers like this manually, meaning a Collective will host an event using Eventbrite, and then the ticket sale proceeds get sent to one of our fiscal hosts, and we credit it to the Collective balance. Same goes for t-shirt sales through threadless and that kind of thing. If any Collectives want to set this up, just contact support@opencollective.com.

Unfortunately we don't have the developer capacity to build these integrations into the software right now, but I hope we can in the future. If any developers are interested in working on such a project we can consider supporting it (technically and financially).

**Comment by gusaus** [comment]
Meetup.com integration is another one that's been requested. https://github.com/opencollective/opencollective/issues/1515

In addition to integration with event services, we should integrate into CMS platforms like WordPress and Drupal.

If the process for onboarding and incentivizing contributors was clear https://github.com/opencollective/opencollective/issues/2423 https://github.com/opencollective/opencollective/issues/2420, many more users/community members would contribute.
